tests/system/providers/amazon/aws/example_eks_with_fargate_in_one_step.py

```diff
@@ -106,13 +106,15 @@
 
     describe_pod = BashOperator(
         task_id="describe_pod",
-        bash_command=""
-        # using reinstall option so that it doesn't fail if already present
-        "install_aws.sh --reinstall " "&& install_kubectl.sh --reinstall "
-        # configure kubectl to hit the cluster created
-        f"&& aws eks update-kubeconfig --name {cluster_name} "
-        # once all this setup is done, actually describe the pod
-        "&& kubectl describe pod {{ ti.xcom_pull(key='pod_name', task_ids='run_pod') }}",
+        bash_command=f"""
+            install_aws.sh || true;
+            install_kubectl.sh || true;
+            # configure kubectl to hit the cluster created
```

Comment:
You have embedded this comment in the command string now.  I don't know for sure, but I wouldn't expect that would work???

Reply:
This will actually be fine I think.

Triple quote is being used which is going to preserve the formatting, which means new lines will be preserved, which means the comments will be evaluated as a standalone statement and since hash is also comment in bash this will just be ignored.

In fact, I think using triple quotes here is actually the real fix, I suspect you could drop all the semi-colons at the end of the lines here and this would still work. (and conversely the kind of wacky string concat that was being used previously, which didn't preserve any formatting was causing issues).

Reply:
Ah, cool.  Thanks for confirming. :+1: